cloudinit/net/__init__.py

```diff
@@ -372,9 +372,6 @@ def device_devid(devname):
 
 
 def get_devicelist():
-    if util.is_FreeBSD() or util.is_DragonFlyBSD():
```

Comment:
Why was this section removed?

Reply:
because there's now a get_devicelist in networking.py

Reply:
however, this will need an actual refactor, so I'm adding that line back in.